crates/dojo/lang/src/semantics/tests.rs

```diff
@@ -22,6 +22,8 @@ test_file_test!(
         get_models_test_class_hashes: "get_models_test_class_hashes",
 
         spawn_test_world: "spawn_test_world",
+
+        poseidon_hash_string: "poseidon_hash_string",
```

Comment:
```suggestion
```
Can be removed, this file is not used for now and will be deprecated with proc macros.